src/main/java/org/broadinstitute/hellbender/tools/funcotator/vcfOutput/VcfOutputRenderer.java

```diff
@@ -180,13 +182,21 @@ private VCFHeader createVCFHeader() {
 
     /**
      * Creates a {@link String} containing the field names from our {@link VcfOutputRenderer#dataSourceFactories} suitable for putting in the VCF header.
+     *
+     * Gencode annotations are put first and then the rest.
+     *
      * @param dataSourceFactories A {@link List} of {@link DataSourceFuncotationFactory} objects from which to pull field names.
      * @return A {@link String} containing the field names from our {@link VcfOutputRenderer#dataSourceFactories} suitable for putting in the VCF header.
      */
     private static String getDataSourceFieldNamesForHeader(final List<DataSourceFuncotationFactory> dataSourceFactories) {
-        return dataSourceFactories.stream()
+        return dataSourceFactories.stream().sorted(DataSourceUtils::datasourceComparator)
                         .map(DataSourceFuncotationFactory::getSupportedFuncotationFields)
                         .flatMap(LinkedHashSet::stream)
                         .map(Object::toString).collect(Collectors.joining(HEADER_LISTED_FIELD_DELIMITER));
     }
+
+    private static String retrieveSanitizedFuncotation(final Funcotation funcotation, final String manualAnnotationSerializedString) {
+        final String initialString = funcotation.serializeToVcfString(manualAnnotationSerializedString);
+        return StringUtils.replaceEach(initialString, new String[]{",", ";", "=", "\t", "|"}, new String[]{"_%2C_", "_%3B_", "_%3D_", "_%09_", "_%7C_"});
```

Comment:
we don't actually support percent encoding in vcf yet, it's a 4.3 feature and we're only up to 4.2

Reply:
I'm assuming that this comment was for our information.  I just wanted a reversible transform. We will probably have to adjust for VCF 4.3.  @lbergelson Should I take any more steps?

No action (for now).